openreview/venue/invitation.py

```diff
@@ -2024,6 +2027,13 @@ def set_custom_stage_invitation(self):
                     'withForum': '${6/content/noteId/value}'
                 }
             }
+        elif custom_stage_replyto == 'review_revisions':
+            paper_invitation_id = self.venue.get_invitation_id(name=custom_stage.name, prefix='${2/content/replytoSignatures/value}')
+            with_invitation = self.venue.get_invitation_id(name=self.venue.review_stage.name, number='${6/content/noteNumber/value}')
+            reply_to = None
+            edit_readers = [venue_id]
```

Comment:
edit readers are usually just the note readers, but in this case we are not setting note readers in the invitation (since we want to keep the review readers). I set the venueid as the readers of the edit but I am not sure this is correct.

Reply:
can you add the signatures of the edit as reader too?